sphinx/faq.rst

```diff
@@ -42,15 +41,6 @@ General Questions
 
 Specifics
 ^^^^^^^^^
-#. Can I specify a custom sort order for the items in a choicebox?
-
-   No, there is no way to specify a custom order. The reason is that
-   the data must be sorted in order for the "jump to" feature
-   (namely, to jump down in the list by pressing keyboard keys) to work.
-
-   For some tips on how to control the order of items in a choicebox,
-   see this recipe from the Easygui Cookbook.
-
```

Comment:
Yes. The sorting thing was never necessary, and the workaround isn't needed. I think the confusion could have come from the fact that clicking a choice highlights it, but the previously selected choice is still underlined, and using the arrow keys selects a new choice above or below the underlined choice instead of the highlighted one.

Or maybe it was some other issue? Either way, I couldn't seem to recreate the problem this workaround was supposed to be for. And the sorting behavior was all dead code at this point anyway, so I took the code and the documentation for it out.